client/my-sites/checkout/checkout/test/country-specific-payment-fields.js

```diff
@@ -21,23 +24,38 @@ const defaultProps = {
 };
 
 describe( '<CountrySpecificPaymentFields />', () => {
-	test( 'should render', () => {
-		const wrapper = shallow( <CountrySpecificPaymentFields { ...defaultProps } /> );
-		expect( wrapper ).toMatchSnapshot();
```

Comment:
Happy to see this removed 👍 I'm curious if there was a good reason to have it in the first place.